custom/icds_reports/tests/agg_tests/test_export_data.py

```diff
@@ -3810,7 +3813,7 @@ def test_growth_tracker_export(self):
                 [
                     'Export Info',
                     [
-                        ['Generated at', india_now()],
+                        ['Generated at', '16:21:11 15 November 2017'],
```

Comment:
nit: could set the `now` value in `setUpClass`:
```py
    def setUpClass(cls):
        super(TestExportData, cls).setUpClass()
        cls.now = '16:21:11 15 November 2017'
        ...
```
Then you could reference that here instead of duplicating the constant value.
```suggestion
                        ['Generated at', self.now],
```

Reply:
updated!